raft-tech/cfgov-refresh
# deleted starting balance logic

Client does not want to show starting week on transaction header on Calendar view.




## Removals

- removed <div> from details.js file



## How to test this PR

1.  Open app. 
2. Add a starting balance of $100
3.  Add a rent payment for the week after the first week of $800
4.  You should be able to view the _transaction header_ on the first week (green) and the second week (red).
5. You should see no Beginning Balance.


## Screenshots


<img width="150" alt="Screen Shot 2020-08-11 at 12 51 44 PM" src="https://user-images.githubusercontent.com/34319929/89926554-14b1cb00-dbd3-11ea-9a17-b10f8e655020.png">
<img width="150" alt="Screen Shot 2020-08-11 at 12 51 52 PM" src="https://user-images.githubusercontent.com/34319929/89926556-14b1cb00-dbd3-11ea-8e21-801bdf999753.png">
